cmd/system-probe/config/config.go

```diff
@@ -203,6 +204,8 @@ func load(configPath string) (*Config, error) {
 	if !cfg.GetBool("network_config.enabled") && cfg.GetBool("service_monitoring_config.enabled") {
 		log.Info("service_monitoring.enabled detected: enabling system-probe with network module running.")
 		c.EnabledModules[NetworkTracerModule] = struct{}{}
+		// ensure others can key off of this single config value for NPM status
+		cfg.Set("network_config.enabled", true)
```

Comment:
We actually need to know the status NPM separate from USM. This allows parts of the NPM backend to drop data once it is no longer needed.